packages/crud/src/vaadin-crud-grid.js

```diff
@@ -195,12 +221,22 @@ class CrudGrid extends IncludedMixin(Grid) {
           textField.setAttribute('slot', 'filter');
           textField.setAttribute('focus-target', true);
           textField.style.width = '100%';
-          this.noSort && (textField.placeholder = label);
+
+          if (this.noSort) {
+            textField.placeholder = label;
```

Comment:
I understand that this is existing behavior, but I'm wondering if using the label as placeholder to have a visual column header label isn't considered a misuse.

Reply:
Using the placeholder for the header label is a bad practice.